lms/djangoapps/certificates/management/commands/ungenerated_certs.py

```diff
@@ -8,6 +8,7 @@
 from django.core.management.base import BaseCommand, CommandError
 from certificates.models import certificate_status_for_student
 from certificates.queue import XQueueCertInterface
```

Comment:
Is it possible to move this XQueueCertInterface import into certificates.api as well, like you did in regenerate_user.py?